docs/1000.yaml

```diff
@@ -898,12 +956,16 @@ Q942046:
 
 Q943246:
   title: Wedderburn's little theorem
+  decl: littleWedderburn
 
 Q944297:
   title: Open mapping theorem
+  # TODO: this entry points to Wikipedia's disambiguation page => wikipedia should be fixed
+  # Banach open mapping theorem/Banach-Schauder theorem in functional analysis is in mathlib (ContinuousLinearMap.isOpenMap)
 
 Q948664:
   title: Kneser's theorem
+  # TODO: is this in mathlib already?
```

Comment:
This is [in LeanCamCombi](https://github.com/YaelDillies/LeanCamCombi/tree/master/LeanCamCombi/Kneser) (authored by Mantas Baksys and I)